dags/legacy_test/tests/pytorch/nightly/common.libsonnet

```diff
@@ -181,10 +181,13 @@ local volumes = import 'templates/volumes.libsonnet';
         export PATH=~/.local/bin:$PATH
       |||,
       tpuVmExtraSetup: |||
-        # Dependency of acceelerate, unfortunately there is no requirements.txt in acceelerate.
+        if [ -d "$HOME/.local/bin" ] ; then
+          export PATH="$HOME/.local/bin:$PATH"
```

Comment:
Do you know why the `PATH` update in `tpuVmExports` isn't taking effect? Does it need to be `$HOME/etc/etc` instead of `~/etc/etc`?

Reply:
Here is what I found:

> Do you know why the PATH update in tpuVmExports isn't taking effect?

The PATH update in tpuVmExports takes effect in the `run_model` phase. But the error occurs in the setup phase.

> Does it need to be $HOME/etc/etc instead of ~/etc/etc?

~/etc/etc doesn't seem to work. I think it's because of the double quote: https://askubuntu.com/a/1177469